src/documentation/pages/components-wip/date-picker/examples/_base-date-picker.vue

```diff
@@ -0,0 +1,52 @@
+<template lang="pug">
+  <ComponentExample title="Base" id="base-date-picker" :tabs="exampleTabs">
```

Comment:
This image is a comparison between the production and the generated instance.

![image](https://user-images.githubusercontent.com/86958774/179941463-bd01a7f4-5d20-4bc1-8371-976f820915ea.png)

Reply:
@josecarlosgonzalezv The datepicker behavior is the same from what i can see but i can see the difference with the z-index issue here. As we are only consuming chi-date-picker web component in this documentation PR, i believe this issue can be handled seperately from this PR. Please let me know if that's not the case.

Reply:
If you see our current documentation the disabled example has `z-index: 1;`. 

![image](https://user-images.githubusercontent.com/86958774/183612578-74185990-3b52-429e-ae1b-9ee9d4a0798f.png)

The **ComponentExample** has the `additionalStyle` property that supports custom styles, by doing this you can get the same behaviour.

`<ComponentExample title="Disabled" id="date-disabled" :tabs="exampleTabs" additionalStyle="z-index: 1;">`